apisix/discovery/nacos/init.lua

```diff
@@ -54,6 +56,25 @@ local function get_key(namespace_id, group_name, service_name)
     return namespace_id .. '.' .. group_name .. '.' .. service_name
 end
 
+
+local function metadata_contains(host_metadata, route_metadata)
```

Comment:
host_metadata and node.metadata are mixed, it is recommended to unify them as node_metadata